index.bs

```diff
@@ -198,6 +208,14 @@ dictionary which describes modifications to the sanitze operation.
 : dropAttributes
 :: The <dfn>attribute drop list</dfn>  is an [=attribute match list=], which
    determines whether an attribute (on a given element) should be dropped.
+: allowUnknownElements
```

Comment:
I don't think we want a blanket allow-boolean for this case (and the attribute case below). I think we should ask developers to specifically list the elements & attributes they need.

I know this puts an extra onus on e.g., framework authors, but I'd rather do this than provide a footgun that turns this  into a security bug. Happy to debate this further.

Reply:
I'm not sure I agree, honestly. I'll remove them in the next update, but I guess I'll make an attempt at getting them back in again. :)